emotescraper/bmscraper/scraper.py

```diff
@@ -202,13 +203,19 @@ def _callback_fetch_stylesheet(self, response, subreddit=None):
             logger.error("Subreddit not set")
             return
 
-        if not response:
-            logger.error("Failed to fetch css for {}".format(subreddit))
-            return
-
-        self._process_stylesheet_response(response.status_code, response.text, subreddit)
+        if response is not None:
```

Comment:
The original check that I assumed was a None check (and messed stuff up by not continuing processing after my change) was down where we check the status code before my changes, but it doesn't seem to actually do anything. It just logs an error that will be logged again after the next if statement: https://github.com/BTDev/Berrymotes/blob/17d10bb7f6fc7bbddbf2f7e7152b9ed47b6a7167/emotescraper/bmscraper/scraper.py#L199-L204